src/index.js

```diff
@@ -17,6 +17,13 @@ export const SOURCES = {
   R: "Rehatnamas & Panthic Sources"
 };
 
+export const SOURCES_WITH_ANG = {
+  G: SOURCES.G,
+  D: SOURCES.D,
+  B: SOURCES.B,
+  S: SOURCES.S
```

Comment:
what about other granths like Nandlal Jee Vaaran, Bhai Gurdas Singh Jee Vaaran, etc?

Reply:
This is a list of sources, which we can read by Ang.
And I don't think we can read the remaining Granths by Ang. Can we?

Reply:
@bogas04 ping.

Reply:
@tarunsingh5 @ManjotS could you please guide us here? Thanks!

Reply:
By design there are a number of sources without page numbers,  which breaks a few use cases, this could be changed, but not here.